schemas/remote_configuration_v2_template.schema.json

```
@@ -0,0 +1,41 @@
+{
+  "$id": "remote_configuration_v2_template",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "additionalProperties": false,
+  "description": "Config template files stored as object which contain list of conditional gathering rules and list of container logs",
+  "example": {
+    "conditional_gathering_rules": [
+      "conditional_gathering_rules/rule1.json",
+      "conditional_gathering_rules/rule2.json",
+      "conditional_gathering_rules/subdir/rule3.json"
+    ],
+    "container_logs": [
+      "container_log_requests/manual/kube_controller_manager_logs_gatherer.json"
+    ]
+  },
+  "properties": {
+    "conditional_gathering_rules": {
+      "description": "Array of paths to conditional gathering rules",
+      "items": {
+        "description": "Path to conditional gathering rule",
+        "pattern": "^conditional_gathering_rules\\/(.+\\/)*[^\\/]+\\.json$",
```

The contents of `conditional_gathering_rules/` is supposed to be flat because the conditional gathering feature does not handle duplicate requests well. Duplicate requests result in duplicate data, which is undesirable.

I would enforce flat structure in the schema. The schema will be used only internally in this repository. We can always relax it if we needed to allow more complex structure.

P.S. Also simplifying the regex. We don't need to escape `/`.

```suggestion
        "pattern": "^conditional_gathering_rules/[^/]+\\.json$",
```

---

pattern updated as requested